stack_data/core.py

```diff
@@ -46,6 +47,11 @@
 """
 
 
+class BlankLines(Enum):
+    HIDDEN = 1
+    VISIBLE = 2
+    LINE_NUMBER=3
```

Comment:
This name doesn't feel like it explains itself well, especially since multiple blank lines specifically mean no line number gets shown. Maybe `COLLAPSED`?

Reply:
I wasn't entirely satisfied with my choice of `LINE_NUMBER`, but it's the best I could come up with at the time.  I'll make the change to `COLLAPSED`.

Reply:
I've changed it to `SINGLE` with a docstring added describing what each name means.